sql/expression/function/st_length_test.go

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020-2022 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package function
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/expression"
+)
+
+func TestSTLength(t *testing.T) {
+	t.Run("select unit length", func(t *testing.T) {
+		require := require.New(t)
+		f, err := NewSTLength(expression.NewLiteral(sql.LineString{Points: []sql.Point{{X: 0, Y: 0}, {X: 0, Y: 1}}}, sql.LineStringType{}))
+		require.NoError(err)
+
+		v, err := f.Eval(sql.NewEmptyContext(), nil)
+		require.NoError(err)
+		require.Equal(1.0, v)
+	})
+
+	t.Run("select sqrt 2", func(t *testing.T) {
+		require := require.New(t)
+		f, err := NewSTLength(expression.NewLiteral(sql.LineString{Points: []sql.Point{{X: 0, Y: 0}, {X: 1, Y: 1}}}, sql.LineStringType{}))
+		require.NoError(err)
+
+		v, err := f.Eval(sql.NewEmptyContext(), nil)
+		require.NoError(err)
+		require.Equal(1.4142135623730951, v)
```

Comment:
Same comment here -- if your'e going to do something like this, then use math.Sqrt(2) instead